dataworks-aws-k2hb-reconciliation-trimmer.tf

```diff
@@ -0,0 +1,69 @@
+resource "github_repository" "k2hb_reconciliation_trimmer" {
+  name             = "dataworks-aws-k2hb-reconciliation-trimmer"
```

Comment:
resource "github_repository" "k2hb_reconciliation_trimmer" {
   name             = "dataworks-aws-k2hb-reconciliation-trimmer"

these should match?